### ISSUE TITLE
b-form-input type="number" not allowing scientific notation

### ISSUE THREAD

**Issue by sorayadragon** [bootstrap-vue/bootstrap-vue#2247]
Version: v2.0.0-rc.11
Browsers: Chrome, FF & Edge

When using the b-form-input component with type="number", it does not allow the user to enter a number in scientific notation (i.e 1.0e5). When 'e' is entered into the field it clears the value. However, scientific notation does display correctly if passed in by a variable and not user input.

Reproduceable in the documentation (https://bootstrap-vue.js.org/docs/components/form-input/).

**Comment by tmorehouse** [comment]
Try the latest dev branch, which should have this issue addressed.

**Comment by sorayadragon** [comment]
I tried installing the dev branch: `npm install --save https://github.com/bootstrap-vue/bootstrap-vue#dev`

It installed fine, but now I can't import it into my Vue file; the following lines produce errors:

```
import BootstrapVue from 'bootstrap-vue'
import 'bootstrap-vue/dist/bootstrap-vue.css'
Vue.use(BootstrapVue);
```

When will the dev branch be merged into master?

**Comment by TitanFighter** [comment]
@sorayadragon 
Try `import BootstrapVue from 'bootstrap-vue/src/index'`

Regarding css, read [this](https://github.com/bootstrap-vue/bootstrap-vue/blob/58da93b7876e4d24443bc2126f5ecf9ff4503a3a/docs/markdown/intro/README.md#webpack).

**Comment by tmorehouse** [comment]
When using the `dev` branch, you will need to run `yarn build` to generate the `dist` and `es` files.

**Comment by tmorehouse** [comment]
Confirmed this issue is fixed via PRs #2100, #2204 (available in latest `dev` branch)
